service/history/nDCEventsReapplier.go

```diff
@@ -26,18 +26,21 @@ import (
 	ctx "context"
 
 	"github.com/uber/cadence/.gen/go/shared"
+	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/metrics"
+	"github.com/uber/cadence/common/persistence"
 )
 
 type (
 	nDCEventsReapplier interface {
-		reapplyEvents(ctx ctx.Context, msBuilder mutableState, historyEvents []*shared.HistoryEvent) error
```

Comment:
this should be `[]*shared.HistoryEvent` not `*shared.DataBlob` since this function will only be used as part of business logic

Reply:
done